samples/nrf9160/modem_shell/README.rst

```diff
@@ -121,13 +147,25 @@ MoSh command: "curl"
 
 Curl is a command-line tool for transferring data specified with URL syntax.
 It is a part of MoSh and enables you to test the data download with a "standard" tool.
+Note: there is a limitation that file operations are not supported.
```

Comment:
```suggestion
It is a part of MoSh and enables you to test the data download with a "standard" tool.

.. note::
   File operations are not supported.
```

Reply:
Adding this to my next change as cannot commit from github review comment anymore after changes.